package.json

```diff
@@ -8,7 +8,7 @@
     "start": "next start",
     "lint": "next lint",
     "test": "jest",
-    "test:cov": "yarn test --silent --coverage --coverageReporters=text --coverageReporters=text-summary --watchAll=false",
+    "test:cov": "yarn test --silent --coverage --coverageReporters=text --coverageReporters=text-summary --watchAll=false --passWithNoTests",
```

Comment:
Out of curiosity, why did you add this option?

Reply:
To handle this issue: 
<img width="1211" alt="Screenshot 2023-03-22 at 11 35 36" src="https://user-images.githubusercontent.com/66369091/226938086-ed80ada1-28f5-46f9-9522-233f799e7275.png">

Reply:
Currently there are no tests

Reply:
Got it! I guess it's the same as deleting the file.. it's ok for me the way you chose